Cannot install as 'nearleyc@latest' is not in the npm registry

It seems in the latest update, the NPM post-install method tries to use the module `nearleyc` which does not exist.

If you try to install with NPM, this is the error output:

`npm ERR! code 1
npm ERR! path .../node_modules/address-rfc2821
npm ERR! command failed
npm ERR! command sh -c npx nearleyc <grammar.ne -o grammar.js
npm ERR! npm ERR! code E404
npm ERR! npm ERR! 404 Not Found - GET https://registry.npmjs.org/nearleyc - Not found
npm ERR! npm ERR! 404 
npm ERR! npm ERR! 404  'nearleyc@latest' is not in the npm registry.
npm ERR! npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! npm ERR! 404 
npm ERR! npm ERR! 404 Note that you can also install from a
npm ERR! npm ERR! 404 tarball, folder, http url, or git url.
npm ERR! 
npm ERR! npm ERR! A complete log of this run can be found in:
npm ERR! npm ERR!     /Users/nick/.npm/_logs/2021-01-04T22_26_49_965Z-debug.log`

--- a/.npmignore
+++ b/.npmignore
@@ -1,0 +1,15 @@
+.github
+logs
+*.log
+pids
+*.pid
+*.seed
+lib-cov
+coverage
+.grunt
+build/Release
+node_modules
+.lock-wscript
+package-lock.json
+grammar.js
+.eslintrc.yaml

--- a/Changes.md
+++ b/Changes.md
@@ -2,11 +2,12 @@
 ### 1.2.1 - 2021-01-01
 
 - The '+' character was mistakenly missing from atext/atom definition
+- package.json: tell npx the nearlyc package name
+
 
 ### 1.2.0 - 2020-12-25
 
-- Replace regular expression based parser with parser based on
-  <https://nearley.js.org/>
+- Replace regular expression based parser with parser based on <https://nearley.js.org/>
 - Double quote characters are not stripped from the local-part of the address if present.
 - No added escaping is attempted inside quoted-strings
 

--- a/package.json
+++ b/package.json
@@ -27,7 +27,7 @@
     "cover": "NODE_ENV=cov npx nyc --reporter=lcovonly npm run test",
     "lint": "npx eslint index.js test/*.js",
     "lintfix": "npx eslint --fix index.js test/*.js",
-    "postinstall": "npx nearleyc <grammar.ne -o grammar.js",
+    "postinstall": "npx -p nearley nearleyc <grammar.ne -o grammar.js",
     "test": "npx mocha"
   },
   "repository": {

